code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -32,6 +32,17 @@
 /mob/camera/aiEye/Move()
 	return 0
 
+//An AI eyeobj mob cant have a virtualhearer to hear with unless it gets one from a malf module
+/mob/camera/aiEye/Hear(message, atom/movable/speaker, var/datum/language/speaking, raw_message, radio_freq)
+	if(radio_freq) //A FUCKING LIP READING MODULE CANT HEAR RADIOS
+		return
+
+	var/mob/M = speaker
+	if(istype(M))
+		if(M.wear_mask && (wear_mask.body_parts_covered & MOUTH)) //NOR CAN IT READ COVERED LIPS

```

Comment:
Bluh bluh helmet no cover mouth

I wrote a proc for this!
It does assume human - because what else has a mask - but it's `check_body_part_coverage(MOUTH)`